homeassistant/components/econet/climate.py

```diff
@@ -0,0 +1,243 @@
+"""Support for Rheem EcoNet thermostats."""
+import logging
+
+from pyeconet.equipment import EquipmentType
+from pyeconet.equipment.thermostat import ThermostatFanMode, ThermostatOperationMode
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    ATTR_TARGET_TEMP_HIGH,
+    ATTR_TARGET_TEMP_LOW,
+    FAN_AUTO,
+    FAN_HIGH,
+    FAN_LOW,
+    FAN_MEDIUM,
+    HVAC_MODE_AUTO,
+    HVAC_MODE_COOL,
+    HVAC_MODE_FAN_ONLY,
+    HVAC_MODE_HEAT,
+    HVAC_MODE_OFF,
+    SUPPORT_AUX_HEAT,
+    SUPPORT_FAN_MODE,
+    SUPPORT_TARGET_HUMIDITY,
+    SUPPORT_TARGET_TEMPERATURE,
+    SUPPORT_TARGET_TEMPERATURE_RANGE,
+)
+from homeassistant.const import ATTR_TEMPERATURE
+
+from . import EcoNetEntity
+from .const import DOMAIN, EQUIPMENT
+
+_LOGGER = logging.getLogger(__name__)
+
+ECONET_STATE_TO_HA = {
+    ThermostatOperationMode.HEATING: HVAC_MODE_HEAT,
+    ThermostatOperationMode.COOLING: HVAC_MODE_COOL,
+    ThermostatOperationMode.OFF: HVAC_MODE_OFF,
+    ThermostatOperationMode.AUTO: HVAC_MODE_AUTO,
```

Comment:
Can the user change temperature in the auto mode? If so we should use `HVAC_MODE_HEAT_COOL`.

https://github.com/home-assistant/core/blob/f67e8b43690b0836ff8ee60ce90b9e015123badc/homeassistant/components/climate/const.py#L12-L17

Reply:
Yes they can. I'll update that.